doc/modules/tree.rst

```diff
@@ -182,6 +182,29 @@ render these plots inline automatically::
    :align: center
    :scale: 75
 
+Alternatively, the tree can also be exported in textual format with the
+function :func:`export_text`. This method doesn't require the installation
+of external libraries and is more compact:
+
+    >>> from sklearn.datasets import load_iris
+    >>> from sklearn.tree import DecisionTreeClassifier
+    >>> from sklearn.tree.export import export_text
+    >>> iris = load_iris()
+    >>> X = iris['data']
+    >>> y = iris['target']
```

Comment:
It's a `sklearn.utils.Bunch` object, so just `iris.data`, `iris.target`,  `iris.feature_names` will work and might be more readable...

Also I guess one could directly use those in `decision_tree.fit` to save 2 lines..